metadata-ingestion/examples/recipes/dbtManifest_to_datahub.yml

```diff
@@ -0,0 +1,15 @@
+---
+source:
+  type: "dbt_manifest"
```

Comment:
would dbt have other forms of integration as a source? 
wondering if we should just name the source type as dbt

Reply:
Probably not immediately, I've been thinking of some level of merged source.  Where we would pull metadata / lineage from DBT but get more details from the DB. 

The other thought I had was ingesting the runtime data as well to include runtime stats and times.  

IE this model took this long to calculate and was last calculated at time(x) with row_count(y).

But as of right now probably fine to just name dbt.